lib/test/unit/ui/console/testrunner.rb

```diff
@@ -540,13 +540,27 @@ def summary_marker_color
             color("#{@result.status}-marker")
           end
 
+          TERM_COLOR_SUPPORT = /
+            color|  # explicitly claims color support in the name
+            direct| # explicitly claims "direct color" (24 bit) support
+            #{ColorScheme::TERM_256}
+            ^cygwin|
+            ^linux|
+            ^nsterm-bce|
+            ^nsterm-c-.*|
```

Comment:
`.*` is needless.

Reply:
yes it is. removed in an upcoming commit.